### PR TITLE
Move execute methods of the partitioning commands

### PR DESCRIPTION
**Pull request by poncovka**
Move `execute` methods of the partitioning commands from
`pyanaconda.kickstart` to `pyanaconda.storage.execution`.

### CODE REVIEW

File: pyanaconda/kickstart.py [hunk 1]
```diff
@@ -1961,28 +1960,7 @@ def is_harddrive_based(self):
...
-            for part in bootPartitions:
-                if part.mountpoint == "/boot":
-                    part.fstype = storage.default_boot_fstype
-
-            reqs += bootPartitions
-
-        if reqs:
-            log.debug("Applying requirements:\n%s", "".join(map(str, reqs)))
-            autopart.do_reqpart(storage, reqs)
+    pass
```
Comment (jkonecny12) [inline pyanaconda/kickstart.py]: Do we need this class when there is no change in the implementation?

Comment (poncovka) [reply pyanaconda/kickstart.py]: Fixed.

**Comment by poncovka** [comment]
Jenkins, test this please.

**Comment by poncovka** [comment]
Rebased.

**Pull request merged**